src/components/victory-area/victory-area.js

```diff
@@ -12,7 +12,8 @@ const fallbackProps = {
   width: 450,
   height: 300,
   padding: 50,
-  interpolation: "linear"
+  interpolation: "linear",
+  onEnd: identity
```

Comment:
This should not be a prop by itself, it should be attached to the animate prop, or better yet provided as a fallback in transition.js itself as it is with the other callbacks (_i.e._ `before`)